### DISCUSSION TITLE
Bridge integration

### DISCUSSION THREAD

**Discussion by AnnaAntonatouPap** [satijalab/seurat#9053] [Q&A]
Hello,

I would like to ask a general question. I am doing spatial transcriptomics analysis ( similar to Slide-seq) and I want to integrate them with scATACseq data, based on my reading the best idea is to use dictionary learning since I also have a 10x multiome dataset, and use the ST dataset as a reference. I see in the vignette, I can visualize the predicted annotations for the scATACseq on the reference UMAP embedding,I was wondering if I can visualize ATAC features into the spatial plot ( SpatialFeaturePlot)

Thanks in advance
Anna

**Comment by CadenZhao** [comment]
Hello. Did you find solutions? I'm doing similar things.

**Reply by YoukaiFromAccounting** [reply]
@AnnaAntonatouPap @CadenZhao What ATAC features are you interested in visualizing on a cell-specific basis in SpatialFeaturePlot?

**Reply by CadenZhao** [reply]
> @AnnaAntonatouPap @CadenZhao What ATAC features are you interested in visualizing on a cell-specific basis in SpatialFeaturePlot?

Peak count and any features can be computed using peak info, like gene activity, TF/motif activity, etc.

**Reply by YoukaiFromAccounting** [reply]
If your spatial dataset already includes an ATAC assay, you can directly use this feature for visualization, simply by specifying a meta.data column in SpatialFeaturePlot. If the ATAC and spatial datasets are separate, you can try using Seurat’s transfer anchor framework described in the [vignette](https://satijalab.org/seurat/articles/seurat5_atacseq_integration_vignette) to project the ATAC-derived feature onto the spatial dataset, then transferring peak count information from the scATAC-seq cell dataset to the spatial dataset using TransferData.

**Reply by CadenZhao** [reply]
Thanks for your reply!
